pydantic/type_adapter.py

```diff
@@ -113,7 +112,7 @@ class TypeAdapter(Generic[T]):
     A `TypeAdapter` instance exposes some of the functionality from `BaseModel` instance methods
     for types that do not have such methods (such as dataclasses, primitive types, and more).
 
-    Note that `TypeAdapter` is not an actual type, so you cannot use it in type annotations.
+    **Note:** `TypeAdapter` instances are not types, and cannot be used as type annotations for fields.
```

Comment:
Nice, I'll make sure to refer to this when the weekly type adapter issue / question pops up.